api-service/database/licenses.go

```diff
@@ -150,6 +166,47 @@ func (md *MongoDatabase) ListLicenses(full bool, sortBy string, sortDesc bool, p
 	return out, nil
 }
 
+// ListLicenses list licenses
+func (md *MongoDatabase) ListLicenses(sortBy string, sortDesc bool, page int, pageSize int, location string, environment string, olderThan time.Time) ([]interface{}, utils.AdvancedErrorInterface) {
+	//Find the informations
+	cursor, err := md.Client.Database(md.Config.Mongodb.DBName).Collection("hosts").Aggregate(
+		context.TODO(),
+		mu.MAPipeline(
+			FilterByOldnessSteps(olderThan),
+			FilterByLocationAndEnvironmentSteps(location, environment),
+			mu.APUnwind("$features.oracle.database.databases"),
+			mu.APUnwind("$features.oracle.database.databases.licenses"),
+			mu.APProject(
+				bson.M{
```

Comment:
Should not the licenses filtered by Count > 0?

Reply:
Wireframe shows even licenses with count == 0, but I agree with you and I'll add a filter!